src/descriptors/structs.json

```diff
@@ -177,5 +177,68 @@
       }
     ],
     "supertype": "Forecast"
+  },
+  {
+    "struct_name": "CostCoefficient",
```

Comment:
We should make this more generic. This should be something like a PWL function forecast.

Reply:
`PiecewiseCost` ?